Transport/Simple.H

```diff
@@ -405,15 +427,16 @@ struct SimpleTransport
 
           amrex::Real muloc, xiloc, lamloc;
           amrex::Real Ddiag[NUM_SPECIES] = {0.0};
+          amrex::Real chi_mix[NUM_SPECIES] = {0.0};
           transport(
-            wtr_get_xi, wtr_get_mu, wtr_get_lam, wtr_get_Ddiag, T, rho, massloc,
-            Ddiag, muloc, xiloc, lamloc, tparm);
+            wtr_get_xi, wtr_get_mu, wtr_get_lam, wtr_get_Ddiag, wtr_get_chi, T,
+            rho, massloc, Ddiag, chi_mix, muloc, xiloc, lamloc, tparm);
 
           //   mu, xi and lambda are stored after D in the diffusion multifab
           for (int n = 0; n < NUM_SPECIES; ++n) {
             D_out(i, j, k, n) = Ddiag[n];
+            chi_out(i, j, k, n) = chi_mix[n];
```

Comment:
strictly speaking this could also be deactivated if use_soret is false, but probably not a big deal either way